scripts/eslint-pre-commit

```diff
@@ -1,12 +0,0 @@
-#!/bin/bash
-
-git stash -q --keep-index
-
-# Test prospective commit
-git diff-index --cached HEAD --name-only --diff-filter ACMR | egrep '.js$' | xargs -I $(npm bin)/eslint {}
```

Comment:
Congratulations!

Perhaps we can fold it into a new `MAJOR` release of `sinonjs/sinon` that also includes `nise` and removes the already deprecated stuff?